alfresco-webscripts-client/alfresco-webscripts-spring/src/main/java/eu/xenit/alfresco/webscripts/client/spring/http/InsecureSslHttpComponentsClientHttpRequestFactory.java

```diff
@@ -22,4 +25,8 @@ public InsecureSslHttpComponentsClientHttpRequestFactory() {
                 .setSSLHostnameVerifier(new NoopHostnameVerifier())
```

Comment:
Can't the TTL be configured here ?
`.setConnectionTimeToLive(httpProperties.getConnectionTimeToLive(), TimeUnit.MILLISECONDS)`